cmd/spire-agent/cli/run/run.go

```diff
@@ -105,8 +102,6 @@ func (*RunCLI) Run(args []string) int {
 		fmt.Println(err.Error())
 	}
 
-	cli.SetUmask(c.Log, c.umask)
```

Comment:
We still need this call, since it prevents running SPIRE with a dangerous umask setting. What would be appropriate is removing the 2nd argument (and the subsequent code that handles that argument within SetUmask).

Reply:
Oh, I misunderstood.
I will fix it to remove the 2nd argument.

Reply:
Awesome! Thanks.

Reply:
I fix it.
- https://github.com/spiffe/spire/pull/879/commits/e6d27295c064c3a6c5cf9429192d75967448b7f1
- https://github.com/spiffe/spire/pull/879/commits/cca8dcf3d8279ab2059ae90363f495a5e74ab486

Reply:
Please comment if you prefer to rebase commits.